runtime/help/tutorial.md

```diff
@@ -1,33 +1,66 @@
 
-See `> help defaultkeys` for a list an explanation of the default keybindings.
+## Command mode
 
-### Settings
+Press `Ctrl-e` to open micro's command prompt. Typing `help tutorial` will open
+this documentation.
+
+For the rest of the docs `> help tutoral` indicates pressing `Ctrl-e`.
+
+## Default keyboard shortcuts
+
+Enter `> help defaultkeys` for a list of the default keybindings.
+
+## Simple workflow: Move content from one file to another
+
+For this example, we edit micro's own codebase, and move contents between files.
+
+Press `Ctrl-o` and enter filename to open the file in the current view
+(use `Tab` key to help with autocompletion)
+```
+> open internal/config/plugin_manager.go
+```
+Then from the command prompt (`Ctrl-e`) open a second file in a vertical split
+```
+> vsplit internal/config/plugin.go
+```
+Use `Ctrl-w` ("jump to next split" shortcut) to switch to the first file and
+cut the `PluginInfo` structure into clipboard using `Ctrl-x`.
+
+Press `Ctrl-w` again to switch back to second file and paste the clipboard
+content using `Ctrl-v`.
+
+Now press `F2` to save current file and `Ctrl-q` to close it.
+
+To preview the changes, run `git diff` by pressing `Ctrl-b` ("shell mode")
+and entering the command. You will see changes only to the second file
+`plugin.go`, because the first file is not saved yet.
+
+Hit `Ctrl-q` again and micro will prompt if you want to save the first file
+before closing. Press `y` and you're done.
+
+Congratulations with completing your first mouseless tutorial with micro.
+
+## Settings
 
 In micro, your settings are stored in `~/.config/micro/settings.json`, a file
-that is created the first time you run micro. It is a json file which holds all
-the settings and their values. To change an option, you can either change the
-value in the `settings.json` file, or you can type it in directly while using
-micro.
-
-Press Ctrl-e to go to command mode, and type `set option value` (in the
-future, I will use `> set option value` to indicate pressing Ctrl-e). The change
-will take effect immediately and will also be saved to the `settings.json` file
-so that the setting will stick even after you close micro.
-
-You can also set options locally which means that the setting will only have
-the value you give it in the buffer you set it in. For example, if you have two
-splits open, and you type `> setlocal tabsize 2`, the tabsize will only be 2 in
-the current buffer. Also micro will not save this local change to the
-`settings.json` file. However, you can still set options locally in the
-`settings.json` file. For example, if you want the `tabsize` to be 2 only in
-Ruby files, and 4 otherwise, you could put the following in `settings.json`:
+that is created the first time you run micro. You can edit the `settings.json`
+file directly, or you can use `> set option value` command from micro, which
+modifies `settings.json` file too, so that the setting will stick even after
+you close micro.
```

Comment:
I think this sentence is too long which makes it harder to comprehend compared to the original phrasing. It's also missing the note about the modification taking effect immediately (which importantly does **not** happen when modifying the settings file directly).

Reply:
That's was unclear from the previous doc. Let me try to rephrase that.

Reply:
Try the next text. Should be better now.